src/phy/mod.rs

```diff
@@ -17,73 +17,71 @@
 //! An implementation of the [Device](trait.Device.html) trait for a simple hardware
 //! Ethernet controller could look as follows:
 //!
-/*!
-```rust
```

Comment:
This change makes it a real pain in the ass to update large code blocks.

Reply:
Agreed. If we're okay with some extra warnings, I'd be okay with going back to `/*!`.